stackrox/base/deployment/central-deployment.yaml

```diff
@@ -0,0 +1,173 @@
+# Source: stackrox-central-services/templates/01-central-13-deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
can you rename the directories the plural version of therir `kind` values, for example the parent directory for this resource should be `deployments` not `deployment`